base/utf16.jl

```diff
@@ -66,15 +76,190 @@ function encode16(s::AbstractString)
     UTF16String(buf)
 end
 
+function ascii_to_utf16(s::Array{UInt8,1})
+    len::Int = sizeof(s)
+    buf = Array{UInt16,1}(len+1)
+    for i = 1:len
```

Comment:
Ah, I see my mistake now, I'd searched by just searching "julia copy" in Google... thanks!